User Guide Assets Section Part 2. (#1083)
o3de/o3de.org: content/docs/user-guide/assets/pipeline/asset-types.md

---
linkTitle: Asset Types 
title: Asset Types 
description: Asset Types supported by the Asset Pipeline Open 3D Engine (O3DE).
weight: 700
toc: true
---

The table below describes some of the most common source asset types supported by the **Asset Pipeline** in **Open 3D Engine (O3DE)**.

| Asset file | Description  |
| - | - |
| `.bmp`, `.gif`, `.jpg`, `.jpeg`, `.png`, `.tga`, `.tif`, `.tiff` | Image file formats most often used as textures for materials. |
| `.dds` | A specialized compressed image format most often used for normal maps and texture atlases in real-time applications. |

> | `.exr` | High dynamic range images used for Image Based Lighting (IBL) and skyboxes. |

Reviewer: ```suggestion
| `.exr` | High dynamic range (HDR) images used for Image Based Lighting (IBL) and skyboxes. |

```

| `.fbx` | A scene file format that stores complex meshes with texture and color information, material assignments, skin weights, skeletons, animation, levels of detail, and physics colliders. |

> | `.inputbindings` | Binds input from keyboards, gamepads, touchscreens, and other devices to runtime events |

Reviewer: ```suggestion
| `.inputbindings` | Binds input from keyboards, gamepads, touchscreens, and other devices to runtime events. |

```

| `.prefab` | A reusable, spawnable asset that contains entities or other prefabs. |
| `.stl` | A simple mesh format used in computer aided design (CAD) and 3D printing. |
| `.material` | Contains properties and values that define the look of a surface for rendering. |
| `.physicsmaterial` | Contains properties and values that define the physical properties of a surface for simulation. |

> | `.scriptcanvas` | A Script Canvas graph created with Script Canvas Editor&mdash;O3DE's visual scripting tool. |

Reviewer: ```suggestion
| `.scriptcanvas` | A Script Canvas graph created with Script Canvas Editor, which is O3DE's visual scripting tool. |

```

---
linkTitle: Asset Types 
title: Asset Types 
description: Asset Types supported by the Asset Pipeline Open 3D Engine (O3DE).
weight: 700
toc: true
---

> The table below describes some of the most common source asset types supported by the **Asset Pipeline** in **Open 3D Engine (O3DE)**.

Author: This one got lost in the shuffle. The topic in the Appendix needs to be replaced by this, or just removed. About half of the file types in the Appendix list are legacy formats. Some may be still supported for backwards compatibility, some no longer matter to O3DE. This list is incomplete which is why is says "some of the most common". I intended to continue to update this list and a run across additional asset types. We can work on this in a new PR.

> The table below describes some of the most common source asset types supported by the **Asset Pipeline** in **Open 3D Engine (O3DE)**.

Reviewer 2: Got it. Sounds good to me.

> | Asset file | Description  |

Author: It could be done. I think this asset list was actually started before most of the documentation and before I had a clear concept of the Asset Pipeline. We can get this going in a new PR.
